ddtrace/__init__.py

```diff
@@ -56,7 +56,7 @@
 
 __all__ = [
     "patch",
-    "patch_all",
+    "_patch_all",
```

Comment:
This is a breaking change. Customers that once were able to

```py
from ddtrace import patch_all
patch_all()
```

will now get an import error. Or am I missing something? Will this proposed change happen in a major version bump?

Reply:
That's what was originally intended, but now we're going to wait so I'll keep patch_all exposed.